src/wp-includes/rest-api/endpoints/class-wp-rest-sidebars-controller.php

```diff
@@ -141,10 +187,6 @@ public function get_item( $request ) {
 
 		$sidebar = $this->get_sidebar( $request['id'] );
 
-		if ( ! $sidebar ) {
-			return new WP_Error( 'rest_sidebar_not_found', __( 'No sidebar exists with that id.' ), array( 'status' => 404 ) );
```

Comment:
Why do we remove this check?

Reply:
It was remove the `get_item_permissions_check` method, where always should have beenm
https://github.com/WordPress/wordpress-develop/pull/1578/files#diff-d5cf0688236a671f97ce276bb4900a3887b9ee1d2b3f8d4100fe72ad3797d13eR149-R151